common/scripts/install-run.js

```diff
@@ -1,7 +1,27 @@
 "use strict";
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See the @microsoft/rush package's LICENSE file for license information.
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
```

Comment:
Are these new lines auto-generated from the new version?

Reply:
Yes